nservicebus/messaging/send-a-message.md

```diff
@@ -6,12 +6,17 @@ redirects:
 - nservicebus/how-do-i-send-a-message
 ---
 
-To send a message, use the `Send` method on the `IBus` interface, passing as the argument the instance of the message to deliver:
+Prior to V6 sending a message involved using the `Send` method on the `IBus` interface, passing as the argument the instance of the message to deliver. In V6 the equivalent of the `IBus` is the `IBusContext` family of interfaces.
```

Comment:
@timbussmann like this: "Message sending involves using `Send` operation that take an argument of the message to be delivered. Refer to samples for details." ?